README.md

```diff
@@ -12,7 +12,8 @@ The Medical Imaging Server for DICOM is a .NET Core implementation of DICOMweb&t
 
 Azure Health Data Service [DICOM service](https://docs.microsoft.com/en-us/azure/healthcare-apis/dicom/deploy-dicom-services-in-azure) is a managed service and recommended for production deployment.
 
-Review [maintainance guide]( ./docs/resources/dicom-server-maintaince-guide.md) if you want to manage you own deployment. 
+Only the nuget libraries released from this repo are supported.
```

Comment:
The use of "supported" here might be misleading. We don't do official support of the OSS outputs. But I know _exactly_ what you mean (I think).  Could we reword to "Only the Nuget libraries released from this repo are meant to be used in production." (or something like that?)